### PR TITLE
Tweak: Adding packages loading config validation [ED-10249]

### PR DESCRIPTION
**Pull request by rotemee**

### CODE REVIEW

File: core/editor/config-providers/editor-v2-config-provider.php [hunk 1]
```diff
@@ -118,17 +118,14 @@ private function get_packages_data() {
...
-
 					return [
-						$name => [
-							'handle' => $data['handle'],
-							'src' => $data['src'],
-							'deps' => $data['deps'],
-							'i18n' => $data['i18n'],
-							'type' => $type,
-						],
+						$name => array_replace([
```
Comment (StyleShit) [inline core/editor/config-providers/editor-v2-config-provider.php:126]:
```suggestion
						$name => array_replace( [
```

File: core/editor/config-providers/editor-v2-config-provider.php [hunk 2]
```diff
@@ -118,17 +118,14 @@ private function get_packages_data() {
 			$packages_data = apply_filters( 'elementor/editor-v2/packages/config', [] );
 
 			$this->packages_data = Collection::make( $packages_data )
+				->filter( function ( $package_data ) {
+					return isset( $package_data['handle'] );
```
Comment (StyleShit) [inline core/editor/config-providers/editor-v2-config-provider.php:122]:
```suggestion
					return ! empty( $package_data['handle'] );
```

maybe?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: core/editor/config-providers/editor-v2-config-provider.php (hunk 1 above)
Comment (rotemee) [reply core/editor/config-providers/editor-v2-config-provider.php:126]: Done.

File: core/editor/config-providers/editor-v2-config-provider.php (hunk 2 above)
Comment (rotemee) [reply core/editor/config-providers/editor-v2-config-provider.php:122]: Done.

**Review by StyleShit — DISMISSED** [review]

**Review by Nevoss — DISMISSED** [review]

**Review by StyleShit — APPROVED** [review]

**Review by Nevoss — APPROVED** [review]

**Pull request merged**